Documentation/Setup/Config/Index.rst

```diff
@@ -2557,33 +2557,46 @@ sys\_language\_mode
          string
 
    Description
-         Setting various modes of handling localization.
-
+         Configures what system should do when a page is not translated
+         to the requested language.
+         It is only evaluated when `sys_language_uid` is greater than `0`,
+         and the requested page translation is NOT available.
+         Internally this setting influences the value set to the 
+         `$GLOBALS['TSFE']->sys_language_content`.
+         
          The syntax is "[keyword] ; [value]".
 
-         Possible keywords are:
+         **Possible keywords are:**
 
-         [default] - The system will look for a translation of the page (from
-         "Alternative Page Language" table) and if it is not found it will fall
-         back to the default language and display that.
+         [empty] - If not set, and the page is not translated, the system will
+         behave as if the default language was requested.
+         Internally both `$GLOBALS['TSFE']->sys_language_content`
+         and `$GLOBALS['TSFE']->sys_language_uid` whill be set to 0;
 
-         **content\_fallback:** Recommended. The system will always operate
+         **content\_fallback** - Recommended. The system will always operate
          with the selected language even if the page is not translated with a
          page overlay record. This will keep menus etc. translated. However,
          the *content* on the page can still fall back to another language,
-         defined by the value of this keyword, e.g. "content\_fallback ; 1,0"
+         defined by the value of this keyword, e.g. :ts:`content_fallback;1,3,0`
          to fall back to the content of sys\_language\_uid 1 and if that is not
          present either, to default (0).
+         Note that the fallback is on the page level, not on the content level.
+         This means that once a translated page record is found in the fallback
+         chain, system will try to use this language for the rendering even
+         if there is no content translated.
```

Comment:
ah, you;re right :(  I've just tested it again and right now the fallback chain is not changing the page record. I consider this a bug see (https://review.typo3.org/#/c/53307/), but nevertheless for now this sentence has to go away from the docs.